codec/address.go

```diff
@@ -21,10 +21,28 @@ const (
 	maxBech32Size = 90
 )
 
-type Address [AddressLen]byte
+type (
+	LID     [AddressLen]byte // Long ID
```

Comment:
You should not use `AddressLen` to define `LID`...there should be a specific constant for LID